core/testsuite/src/main/java/com/blazebit/persistence/testsuite/AbstractCoreTest.java

```diff
@@ -76,24 +77,7 @@ protected CriteriaBuilderConfiguration configure(CriteriaBuilderConfiguration co
     }
     
     protected Set<String> getRegisteredFunctions() {
```

Comment:
Is there any need for exposing this anymore? I think the method can be removed since it shouldn't be overridden anymore.